ui/desktop/app/templates/scopes/scope/projects/targets/target/sessions.hbs

```diff
@@ -1,6 +1,7 @@
 <Rose::Table as |table|>
   <table.body as |body|>
-    {{#each this.selfActiveSessions as |session|}}
+    {{this.sessions.length}} / {{this.cancelableUserSessions.length}}
```

Comment:
<img width="166" alt="Screen Shot 2021-02-11 at 17 09 39" src="https://user-images.githubusercontent.com/111036/107704907-f749e080-6c8b-11eb-9285-1fe00542e743.png">
 It'd be great to have a label here to explain these numbers. Something like: '2 out of 4 active sessions' or 'Active Sessions: 2. Total Sessions: 4'.

Reply:
Oops.  This was debug output.  Removed.